src/components/widgets/status/StatusTab.vue

```diff
@@ -405,4 +395,18 @@ export default class StatusTab extends Mixins(StateMixin, FilesMixin, ToolheadMi
     direction: rtl;
     text-align: left;
   }
+
+  .progress-button {
+    width: 90px !important;
+    height: 90px !important;
+    border-radius: 50%;
+    overflow: hidden;
+  }
+
+  .progress-button-image {
+    max-width: 71px;
+    max-height: 71px;
```

Comment:
With a 71px image, I've noticed the thumbnail is shifted 1px to the right and bottom, so I'd imagine this will need to be an even number:
![image](https://github.com/fluidd-core/fluidd/assets/25269274/07f6061e-2b8e-44bf-9fdd-4916ea5e151f)
![image](https://github.com/fluidd-core/fluidd/assets/25269274/14391ca1-73e6-4d89-8c24-b706ff395820)

Reply:
Very nice catch!! Took me a bit to repro this as I normally use a 4K 200% scale monitor and it worked fine there, only when I moved to the 1080p 100% scale one I could see the problem!